preupg/settings.py

```diff
@@ -11,37 +11,37 @@
 defenc = "utf-8" if sys.getdefaultencoding() == "ascii" else sys.getdefaultencoding()
 
 # dir where results of analysis are stored
-result_dir = os.path.join("/root", prefix)
+assesment_results_dir = os.path.join("/root", prefix)
 
+results_postfix = "-results"
 # Dir where tar balls are placed
-tarball_result_dir = result_dir+"-results"
+tarball_result_dir = assesment_results_dir + results_postfix
+
+xccdf_template = "xccdf_template.xml"
 
 # base name of XML and HTML file with results
-result_name = "result"
+result_prefix = "result"
 
-tarball_base = result_name + 's'
+tarball_base = result_prefix + 's'
 tarball_prefix = "preupg_"
 tarball_name = tarball_prefix + tarball_base + "-{0}"
 
-xml_result_name = result_name + '.xml'
-html_result_name = result_name + '.html'
+xml_result_name = result_prefix + '.xml'
+html_result_name = result_prefix + '.html'
 
-# base name of custom xsl stylesheet
 xsl_sheet = "xccdf-report.xsl"
 
-old_xsl_sheet = "preup.xsl"
-
 share_dir = "/usr/share"
 # sources delivered by preupgrade assistant package
 source_dir = os.path.join(share_dir, prefix)
```

Comment:
I agree, however I would get rid of the "premigrate" stuff in another branch/pull request. It would be too much changes for this one pull request.